tasks/generate_data.ex

```diff
@@ -71,26 +78,28 @@ defmodule Mix.Tasks.Countriex.GenerateData do
   end
 
   defp parse_geo(geo_data, _type) when geo_data == %{}, do: %Geo{}
+
   defp parse_geo(geo_data, %Country{}) do
     %Geo{
-      latitude:       geo_data.latitude |> to_float,
-      latitude_dec:   geo_data.latitude_dec |> to_float,
-      longitude:      geo_data.longitude |> to_float,
-      longitude_dec:  geo_data.longitude_dec |> to_float,
-      max_latitude:   geo_data.max_latitude |> to_float,
-      max_longitude:  geo_data.max_longitude |> to_float,
-      min_latitude:   geo_data.min_latitude |> to_float,
-      min_longitude:  geo_data.min_longitude |> to_float,
+      latitude: geo_data.latitude |> to_float,
+      latitude_dec: geo_data.latitude_dec |> to_float,
+      longitude: geo_data.longitude |> to_float,
+      longitude_dec: geo_data.longitude_dec |> to_float,
+      max_latitude: geo_data.max_latitude |> to_float,
+      max_longitude: geo_data.max_longitude |> to_float,
+      min_latitude: geo_data.min_latitude |> to_float,
+      min_longitude: geo_data.min_longitude |> to_float
```

Comment:
for consistency with other edits, maybe also

```suggestion
      latitude: geo_data.latitude |> to_float(),
      latitude_dec: geo_data.latitude_dec |> to_float(),
      longitude: geo_data.longitude |> to_float(),
      longitude_dec: geo_data.longitude_dec |> to_float(),
      max_latitude: geo_data.max_latitude |> to_float(),
      max_longitude: geo_data.max_longitude |> to_float(),
      min_latitude: geo_data.min_latitude |> to_float(),
      min_longitude: geo_data.min_longitude |> to_float()
```